### ISSUE TITLE
545407DF - Rockstar Games Presents Table Tennis

### ISSUE THREAD

**Issue by Blackbird88** [xenia-project/game-compatibility#211]
[Marketplace](http://marketplace.xbox.com/en-US/Product/Rockstar-Table-Tennis/66acd000-77fe-1000-9115-d802545407df)

* Crashes when loading the ISO so there is no log sadly

**Comment by TranzRail** [comment]
I've had issues loading ISO's as well. I'd recommend trying to load default.xex directly after extracting the contents of the ISO using an appropriate tool.

**Comment by Blackbird88** [comment]
Hmm that's not a bad idea. Extracting them isn't problem for me.

**Comment by Blackbird88** [comment]
Now it shows black screen and assert.

**Comment by Blackbird88** [comment]
Finally the game generates log.

**Comment by MarioSonic2987** [comment]
Any change?

**Comment by Blackbird88** [comment]
No.

**Comment by shoopdahoop22** [comment]
please make this game work asap.

i need the definitive table tennis experience on my pc.

**Comment by lediszep** [comment]
Any changes?

**Comment by Blackbird88** [comment]
If there were any I'd post them.

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/ba7dc6b2d7187a233d7175c75994828b498bec58)

(state-crash-guest)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2172564/xenia.zip)

**Comment by reditec** [comment]
Guest is still crashing

**Comment by MattiaPell** [comment]
Still crashing

**Comment by montana013** [comment]
Still don't work ? Why?
[url=https://ibb.co/ZcfLjMs][img]https://i.ibb.co/Dw1VJRX/20220605-180854.jpg[/img][/url]

**Comment by lee649** [comment]
![1674826332869](https://user-images.githubusercontent.com/120994656/215101802-39e8d2ea-9b12-45a7-bd3e-a1a88e5157d0.jpg)
The Egyptian player So layman has weird black shadow both in character selecting menu and when playing with him.Running with boma's patch,both 1440p and 720p have this bug.

**Comment by ahmedelakkad** [comment]
anyone found solutions?

**Comment by pingpongrock** [comment]
It crashes for me, any idea on how to make it working?

**Comment by ahmedelakkad** [comment]
> ![1674826332869](https://user-images.githubusercontent.com/120994656/215101802-39e8d2ea-9b12-45a7-bd3e-a1a88e5157d0.jpg) The Egyptian player So layman has weird black shadow both in character selecting menu and when playing with him.Running with boma's patch,both 1440p and 720p have this bug.

How did you get it to run?

**Comment by ahmedelakkad** [comment]
Can I understand why are you deleting comments? the issue is not closed and everyone here is asking for help.

**Comment by Margen67** [comment]
> Can I understand why are you deleting comments? the issue is not closed and everyone here is asking for help.

This is for reporting compatibility. If you want help ask on the Discord server: https://discord.gg/Q9mxZf9

**Comment by lee649** [comment]
everything works fine,the vibration,the 1440p gaming experience,except for Mr.Solayman....
![xenia_canary exe Screenshot 2023 05 05 - 18 48 38 17](https://user-images.githubusercontent.com/120994656/236438864-6c54eb18-e432-4b33-aa6c-e86dbfd73b2a.png)

**Comment by lee649** [comment]
This may solve your problem
https://github.com/xenia-canary/game-patches
![Desktop Screenshot 2023 05 05 - 18 54 00 12](https://user-images.githubusercontent.com/120994656/236439963-6400ef88-0e9d-483d-9879-abbe6e6b2ac3.png)

**Comment by lee649** [comment]
another clearer screenshot of glitchy Solayman
![xenia_canary exe Screenshot 2023 05 05 - 18 56 15 93](https://user-images.githubusercontent.com/120994656/236440296-37dda268-a519-4b7e-813f-9594d59dbf53.png)

**Comment by ahmedelakkad** [comment]
> This may solve your problem https://github.com/xenia-canary/game-patches ![Desktop Screenshot 2023 05 05 - 18 54 00 12](https://user-images.githubusercontent.com/120994656/236439963-6400ef88-0e9d-483d-9879-abbe6e6b2ac3.png)

Thanks will try it!

**Comment by ahmedelakkad** [comment]
> This may solve your problem https://github.com/xenia-canary/game-patches ![Desktop Screenshot 2023 05 05 - 18 54 00 12](https://user-images.githubusercontent.com/120994656/236439963-6400ef88-0e9d-483d-9879-abbe6e6b2ac3.png)

Same issue unfortunately, black screen after rockstar games presents

**Comment by lee649** [comment]
> > This may solve your problem https://github.com/xenia-canary/game-patches ![Desktop Screenshot 2023 05 05 - 18 54 00 12](https://user-images.githubusercontent.com/120994656/236439963-6400ef88-0e9d-483d-9879-abbe6e6b2ac3.png)
> 
> Same issue unfortunately, black screen after rockstar games presents

you should keep the patch enabled both in the patch itself and xenia-canaryconfig.
otherwise,it might be some compatibility issue(s) of your pc

**Comment by MultiKoopa** [comment]
Might be a longshot, but... anybody have any idea why the game just absolutely will not get to the title screen on Steam Deck? It's like the patch isn't enough; regardless of DX12 or Vulkan being used. Some issue with Vulkan and this game in particular, maybe?

**Comment by lee649** [comment]
> Might be a longshot, but... anybody have any idea why the game just absolutely will not get to the title screen on Steam Deck? It's like the patch isn't enough; regardless of DX12 or Vulkan being used. Some issue with Vulkan and this game in particular, maybe?

I guess the emulator runs better on an Nvidia graphics card than an APU.and a Zen 2 APU on a steam deck can't run the game above 60 fps even if you launched the game successfully.

**Comment by MultiKoopa** [comment]
> > Might be a longshot, but... anybody have any idea why the game just absolutely will not get to the title screen on Steam Deck? It's like the patch isn't enough; regardless of DX12 or Vulkan being used. Some issue with Vulkan and this game in particular, maybe?
> 
> I guess the emulator runs better on an Nvidia graphics card than an APU.and a Zen 2 APU on a steam deck can't run the game above 60 fps even if you launched the game successfully.

I don't care about 90 fps, just if it's possible to actually play it

**Comment by lee649** [comment]
maybe you cant on a steam deck,sadly

**Comment by lee649** [comment]
run the game using xenia_canary **with the patch enabled.** https://github.com/xenia-canary/game-patches
Make sure that your CPU is capable.for any graphics card better than 3060ti,try running at 1440p,which makes the game looking much better(you wont believe that it's a **2006** game).
Known bug:"Black shadow" always appears around Egyptian player Solayman when he is **rendered**.

**Comment by MultiKoopa** [comment]
Yes the patch is applied. it just refuses to get past the title screen on Steam Deck. Also, those pictures don't work.

**Comment by lee649** [comment]
> Yes the patch is applied. it just refuses to get past the title screen on Steam Deck. Also, those pictures don't work.

dont count on it:(

**Comment by MultiKoopa** [comment]
> > Yes the patch is applied. it just refuses to get past the title screen on Steam Deck. Also, those pictures don't work.
> 
> dont count on it:(

don't count on what??

**Comment by lee649** [comment]
can anyone fix gamespeed when vsync disabled

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/458
